### DISCUSSION TITLE
View `edit` vs. `update`

### DISCUSSION THREAD

**Discussion by krystof-k** [avo-hq/avo#2601] [Help] [answered]
I had a condition like this:

```ruby
field :name,
  as: :text,
  disabled: -> { view == :edit },
```

which results on disabled field on edit view, but when validation fails, it stops being disabled because the view changes from `edit` to `update`.

Is this intentional? Is there any actual difference between `edit` and `update`? If so it would be nice it to have a mention in the docs about it (where I haven't found any).

**Comment by adrianthedev** [comment, marked as answer]
Replied and tracking on https://github.com/avo-hq/avo/issues/2605
